test/addons-napi/test_async/test.js

```diff
@@ -1,12 +1,30 @@
 'use strict';
 const common = require('../../common');
 const assert = require('assert');
+const child_process = require('child_process');
 const test_async = require(`./build/${common.buildType}/test_async`);
 
+const testException = 'test_async_cb_exception';
```

Comment:
we typically just say `child` and check `process.argv[2] === 'child'` directly ;)